cwatm/hydrological_modules/water_quality/erosed.py

```diff
@@ -39,15 +40,107 @@ class waterquality_erosed(object):
+
+        #print('tov:', np.nanmean(tov, axis=1))
+        #print('tch_min:', np.nanmin(tch), 'tch_max:', np.nanmax(tch), 'tch_mean', np.nanmean(tch))
+        #print('tconc:', np.nanmean(self.var.tconc, axis=1))
+        print('chanman min:', np.nanmin(self.var.chanMan), 'max:', np.nanmax(self.var.chanMan), 'mean:', np.nanmean(self.var.chanMan))
+        manningsFactor = loadmap('manningsN')
+        print('manningsFactor channel calibrated', manningsFactor)
+        channelmanning= loadmap('chanMan')
+        print('chanman uncalibrated .nc input min,max,mean', np.nanmin(channelmanning), np.nanmax(channelmanning),np.nanmean(channelmanning))
+        print('channel gradient min:', np.nanmin(self.var.chanGrad), 'max:', np.nanmax(self.var.chanGrad), 'mean:',
+              np.nanmean(self.var.chanGrad))
+        print('channel lenght min:', np.nanmin(self.var.chanLength), 'max:', np.nanmax(self.var.chanLength), 'mean:',
+              np.nanmean(self.var.chanLength))
+
 
     def dynamic(self):
         """
         Dynamic part of EROSED module
         """
-        print("erosed dyn")
\ No newline at end of file
+        '''
+        # Modified Universal Soil Erosion (MUSLE) for sediment yield
+        # M_(in_land)  = 11.8×〖(Q_surf*q_peak*A_grid)〗^0.56×K×C×P×LS*f_cfr
+        # 11.8 & 0.56 -> calibration parameters, call from settingsfile
+        # Q_surf surface runoff volume in mm from cwatm
+        # q_peak...peak runoff rate (m3/s); a_tc*Q_surf*Agrid/3.6*t_conc
+            #a_tc...frac. of daily rain falling in time of concentration
+            # t_conc...time of concentration for grid (model variable) hour
+        #if self.var.a
+        #self.var.runoffEnergyFactor = self.var.sum_directRunoff * 2
+        '''
+        self.waterquality_vars.dynamic()
+        tov = divideArrays(np.power(self.var.lsFactor, 0.6) * np.power(self.var.manOverland, 0.6),
+                           18 * np.power(self.var.tanslope, 0.3))
+        tch = self.var.travelTime / (60 * 60)  # converted from seconds to hours
+        self.var.tconc = tov + tch  # [hours]
+        # a05 load dummy value: fraction of daily rain falling in the half-hour highest intensity
+        # if time series read netcdf2
+        a05 = np.maximum(np.minimum(loadmap('a05'), 1.0), 0.0) #must be a fraction between 0 and 1
+
+        # atc : fraction of rain falling in the time of concentration
+        self.var.atc = 1 - np.exp(2 * self.var.tconc * np.log(1 - a05))
+
+        # qpeak: peak runoffrate m3/s
+        self.var.qpeak = divideArrays(self.var.atc * self.var.directRunoff[0:4] * 1000 * self.var.cellArea, 3.6 * self.var.tconc) #[m3s-1]
+
+        # MUSLE: sediment yield per day and grid in [1000 kg]
+        self.var.sedYieldLand = 11.8 * np.power(self.var.directRunoff[0:4] * self.var.qpeak * self.var.cellArea, 0.56) * self.var.kFactor * self.var.cFactor * self.var.lsFactor * self.var.fcfr
```

Comment:
What is the 11.8 there?

Reply:
11.8 and 0.56 are site dependent coefficients. 
They are now loaded from the settings file as 'a' and 'b' in the equation via loadmap()